documentation/release-process.md

```diff
@@ -46,6 +70,7 @@ The nightly image is `mcr.microsoft.com/dotnet/nightly/monitor`. The tag list is
 1. Fix issues for the release in the release branch. Backport fixes to `main` branch and other prior release branches as needed.
 1. Invoke [build](<#Build Release Branch>) pipeline as needed.
 1. After successful build, test changes from [dotnet-tools](https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json) feed. Images from the `nightly` branch of the `dotnet-docker` repository will be recreated the next day after the successful build of the release branch.
+1. Bump the version number in the `main` branch. [Example Pull Request](https://github.com/dotnet/dotnet-monitor/pull/1560). 
```

Comment:
This should be moved to the "Merge to Release Branch" section as the last step. The main branch should have it's version update very shortly after completing the merge to the release branch in order to avoid having the main branch interfere with versioning of the release branch.